lxd/storage/drivers/driver_zfs.go

```diff
@@ -34,7 +34,7 @@ var zfsDefaultSettings = map[string]string{
 	"setuid":     "on",
 	"exec":       "on",
 	"devices":    "on",
-	"acltype":    "posixacl",
+	"acltype":    "posix", // acltype "posix" is equivalent to "posixacl". Set as "posix" to avoid setting it multiple times on ensureInitialDatasets.
```

Comment:
Mhh I am wondering if it would be helpful also adding a link to the zfs docs? https://openzfs.github.io/openzfs-docs/man/master/7/zfsprops.7.html#acltype.
It makes it clear that `posixacl` is an alias for `posix`.

Reply:
I think it is not necessary for such a simple concept like an alias.

Reply:
Maybe you can put it in the commit message for reference.